### DISCUSSION TITLE
Warn player if there is no free block section on ride open

### DISCUSSION THREAD

**Discussion by xzyxzy** [OpenRCT2/OpenRCT2#15666] [Feature requests]
OS: Windows 7 x64
Build: OpenRCT2, v0.0.3-develop build 1478 (c89561a)
Screenshot: http://i.imgur.com/vtSmibQ.png

Guest doesn't moves until you go in costruction mode.
Trains don't move in all modes.
But if I remove block brake after tiny lift (1), it works, or If I move it closer to lift it says "Can't test" (2).
1: http://i.imgur.com/BplMNh6.png
2: http://i.imgur.com/OgXOSsz.png
Savefile: https://app.box.com/s/8p83cmw1fvlm2ro2s1vljnyolrvyl9ay

**Comment by IntelOrca** [comment]
Could someone confirm if the rollercoaster works in this saved game in vanilla RCT2?

**Comment by xzyxzy** [comment]
Seems like same behaviour in vanilla.

**Comment by IntelOrca** [comment]
I guess its getting confused with the block brakes and setup of the trains.

**Comment by Gymnasiast** [comment]
One train is covering two block sections: one regular block brake and one cable lift top section. Because of that, no free blocks are left for any train to move it. (In a block system, you always need one free block in order to be able to move.)

Personally I would call it bad track design and not a bug.

**Comment by IntelOrca** [comment]
@Gymnasiast No advice, no feedback or warning message given by the game to suggest this either.

**Comment by Gymnasiast** [comment]
The game doesn't warn you your hills are too high or your dinghy will explode going over that hill either.

**Comment by IntelOrca** [comment]
The game does warn you about other track configuration problems, like block brakes after another or before a station, non-complete track or more than one pull lift. I am going to make this a feature request to warn the player if there is no free block immediately upon starting the ride.
